### ISSUE TITLE
bug in a browser:Uncaught ReferenceError: process is not defined. 

### ISSUE THREAD

**Issue by masx200** [highlightjs/highlight.js#3222]
bug in a browser:

https://github.com/highlightjs/highlight.js/blob/5dc5176c5c1728ce950fba75a40f55881b689f83/tools/build_node.js#L37

Uncaught ReferenceError: process is not defined

You should use "console.warn" when running in a browser.

**Comment by joshgoebel** [comment]
Can you provide a reproducible example or JSFiddle?  That code should never even be run in a web browser... it's solely for Node if you import the files improperly (would double extensions)...

**Comment by masx200** [comment]
```js

import hljs from "highlight.js/lib/core.js";


import javascript from "highlight.js/lib/languages/javascript.js";

export default hljs;
hljs.registerLanguage("javascript", javascript);
```

**Comment by joshgoebel** [comment]
It should be imported with out any extension (so the dynamic exports can do their thing), which is what the warning says.

```
import javascript from "highlight.js/lib/languages/javascript";
```

**Comment by masx200** [comment]
"d:\Documents\github\markdown-reader\node_modules\highlight.js\lib\languages\javascript.js.js"

```js
function emitWarning() {
    if (!emitWarning.warned) {
      emitWarning.warned = true;
      process.emitWarning(
        'Using file extension in specifier is deprecated, use "highlight.js/lib/languages/javascript" instead of "highlight.js/lib/languages/javascript.js"',
        'DeprecationWarning'
      );
    }
  }
  emitWarning();
    module.exports = require('./javascript.js');
```

**Comment by masx200** [comment]
I should be able to use highlight.js in the browser correctly.

**Comment by masx200** [comment]
I upgraded highlight.js from 10.7.2 to 11.0, and then it crashed.

**Comment by joshgoebel** [comment]
As I said, you need to remove the `.js` from your import:

```js
import javascript from "highlight.js/lib/languages/javascript"; // NO .js extension
```

**Comment by joshgoebel** [comment]
Resolved with release of 11.0.1.
